README.md

```diff
@@ -52,6 +57,20 @@ InputFileName=users_list.csv
 OutputPath=C:\quotas
 OutputDir=outputs
 OutputFileName=result.csv
+
+[Resolve-PendingMigrations]
+InputFileName=input_Resolve-PendingMigrations.csv
+OutputDir=Resolve-PendingMigrations
+OutputFileName=output_Resolve-PendingMigrations.csv
+ItemCountThreshold=100
+
+[Get-GroupsReport]
+FirstCompanyIdentifier=compA
+FirstCompanyName=CompanyA
+SecondCompanyIdentifier=compB
+SecondCompanyName=CompanyB
+CompanyIdentifierAttribute=Department
```

Comment:
Line #35 should be updated too. Do you plan to add sample config values for all the scripts?

Reply:
Nope, not for all scripts. We can thus say "for several scripts".

Reply:
Ok